server/models/migrations/v0.6.0-v0.7.0/migrate.sql

```diff
@@ -55,8 +55,6 @@ CREATE TABLE `rubric_payroll` (
 -- For some reason, MariaDB is being a pain with this.  Skipping it for now.
 -- ALTER TABLE `offday` DROP CONSTRAINT `offday_1`;
 -- ALTER TABLE `offday` ADD UNIQUE `offday_1` (`date`);
-
-ALTER TABLE `inventory` ADD COLUMN `is_broken` TINYINT(1) NOT NULL DEFAULT 0;
```

Comment:
While it doesn't hurt anything currently, the `migrations/*` scripts are for updating old databases and usually shouldn't be edited.  It's to help us upgrade old versions of BHIMA in production (if we are ever running more than 2)